package.json

```diff
@@ -3,29 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "^11.10.5",
-    "@emotion/styled": "^11.10.5",
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
-    "axios": "^0.27.2",
-    "jquery": "^3.7.0",
+    "axios": "^1.6.5",
     "lightbox2": "^2.11.3",
     "notistack": "^3.0.1",
     "react": "^18.0.2",
     "react-dom": "^18.0.2",
     "react-helmet": "^6.1.0",
-    "react-image-crop": "^10.1.5",
+    "react-image-crop": "^11.0.4",
```

Comment:
Just to be sure—have you checked that this did not break adding new tags? This release changes the state of overflow, and might have some unforeseen consequences.

Reply:
Adding new tags works just fine